charts/ccd-api-gateway-web/values.yaml

```diff
@@ -22,12 +24,12 @@ nodejs:
     PROXY_PAY_BULKSCAN: http://ccpay-bulkscanning-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     PROXY_PRINT_SERVICE: http://ccd-case-print-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     PROXY_MV_ANNOTATIONS_API_URL: https://em-anno-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    PROXY_REFDATA: http://rd-professional-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   keyVaults:
     ccd:
       secrets:
         - ccd-api-gateway-oauth2-client-secret
-        - ccd-gw-s2s-secret
```

Comment:
is this not same key as https://github.com/hmcts/ccd-api-gateway/pull/290/files#diff-1031eeab417cd8afe77e7e005717d38dR75 ?

Reply:
they haven't used this key in their application code. Instead the key that is used in the app code is `microservicekey-ccd-gw`